packages/Search/examples/point_clouds/leaf_cloud.txt

```diff
@@ -0,0 +1,882 @@
+881
```

Comment:
Is this the input we want? I don't care which one we use but I think you said you wanted another one

Reply:
Correct we have smaller ones now.  Will replace.

Reply:
Done.